cpp/unreal_plugins/SpEngine/Source/SpEngine/SpEngine.cpp

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#include "SpEngine/SpEngine.h"
+
+#include <memory>                  // std::make_unique, std::unique_ptr
+
+#include <Modules/ModuleManager.h> // IMPLEMENT_MODULE
+
+#include "SpCore/Assert.h"
+#include "SpCore/Config.h"
+#include "SpCore/Log.h"
+#include "SpCore/Rpclib.h" // rpc_server
```

Comment:
Comment not needed when referring to our own code. And it is incorrect anyway, because the type being used in this file is `rpc::server`.